### DISCUSSION TITLE
"*" shouldn't be part of the tab name

### DISCUSSION THREAD

**Discussion by gwankyun** [doublecmd/doublecmd#1890] [Suggestions]
Double Commander
Version: 1.1.18 gamma
Revision: 568
Commit: c45365d1b
Build date: 2024/09/21
Lazarus: 3.5.0.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 11 23H2 x86_64

---

It is useful in "Show Tabs List".

**Comment by alexx2000** [comment]
I can remove it from tab-list.

**Reply by gwankyun** [reply]
Thank you very much.

**Comment by alexx2000** [comment]
Done.
